scripts/constituents.py

```diff
@@ -276,17 +276,28 @@ def write_constituent_routines(self, outfile, indent, suite_name, err_vars):
         for evar in err_vars:
             evar.write_def(outfile, indent+1, self, dummy=True)
         # end for
+        # Figure out how many unique (non-tendency) constituent variables we have
+        const_num = 0
+        for std_name, var in self.items():
+            if 'tendency_of' not in std_name:
```

Comment:
Can tendency variables be `<ABC>_tendency_of_<DEF>` or should this be `not std_name.startswith('tendency_of_')`?

Reply:
good call, by this point, we will only have "tendency_of_*" variables - updated.